eng/common/matrix/functions.ps1

```diff
@@ -0,0 +1,345 @@
+Set-StrictMode -Version "4.0"
+
+function CreateDisplayName {
```

Comment:
I tend to put write functions like `function CreateDisplayName([string]$parameter, [hashtable]$displayNames)` and put the `{` on a new line for larger scopes. While what you have is correct I've never liked using the extra verbose param for functions.